tests/macros/APISpecSidebar.test.js

```diff
@@ -271,44 +321,43 @@ function checkResult(html, config) {
 }
 
 /**
-* Call the macro for each of three locales,
-* using the given config, and check the result.
-*/
+ * Call the macro for each of three locales,
+ * using the given config, and check the result.
+ *
+ * @param {TestConfig} config
+ */
 function testMacro(config) {
     for (const locale of ['en-US', 'fr', 'ja']) {
         let testName = `${config.name}; locale: ${locale}`;
-        itMacro(testName, function(macro) {
+        itMacro(testName, async macro => {
             config.locale = locale;
             macro.ctx.env.locale = locale;
             // Mock calls to MDN.subpagesExpand
-            macro.ctx.page.subpagesExpand = jest.fn((page) => {
+            macro.ctx.page.subpagesExpand = jest.fn(async page => {
                 return config.subpages;
             });
-            return macro.call(config.argument).then(function(result) {
-                checkResult(result, config);
-            });
-
+            const result = await macro.call(config.argument);
+            checkResult(result, config);
         });
     }
 }
 
-describeMacro('DefaultAPISidebar', function() {
-
+describeMacro('APISpecSidebar', () => {
     // Set any fixtures that don't change from one
     // test to another
-    beforeEachMacro(function(macro) {
+    beforeEachMacro(macro => {
         // env.slug only has to be truthy
         macro.ctx.env.slug = 'not undefined';
         // Mock calls to L10n-Common and GroupData
         const originalTemplate = macro.ctx.template;
-        macro.ctx.template = jest.fn( async (name, ...args) => {
-            if (name === "L10n:Common") {
-                return commonl10nFixture;
+        macro.ctx.template = jest.fn(async (name, ...args) => {
+            if (name === 'L10n:Common') {
+                return commonL10nFixture;
             }
-            if (name === "GroupData") {
+            if (name === 'GroupData') {
                 return groupDataFixture;
             }
-            return await originalTemplate(name, ...args);
+            return originalTemplate(name, ...args);
```

Comment:
`return await` is unnecessary and can lead to degraded performance.

## See also:
- https://eslint.org/docs/rules/no-return-await